### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rushi-173**
This is pull request to merge to main.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# review-demo
\ No newline at end of file
+# review-demo
+
+Changes done to dev branch
```
Comment (rushi-173) [inline README.md]: give full stop.

**Comment by theshivamverma** [comment]
Add one more line

**Comment by pragyasabharwal** [comment]
Good to go

**Review by rushi-173 — COMMENTED** [review]
Thats it.

File: README.md (hunk 1 above)
Comment (rushi-173) [reply README.md]: full stop is not needed.

**Pull request merged**